addons/account/models/account_invoice.py

```diff
@@ -255,12 +255,14 @@ def _compute_payments(self):
     state = fields.Selection([
             ('draft','Draft'),
             ('open', 'Open'),
+            ('in_payment', 'In Payment'),
             ('paid', 'Paid'),
             ('cancel', 'Cancelled'),
         ], string='Status', index=True, readonly=True, default='draft',
         track_visibility='onchange', copy=False,
         help=" * The 'Draft' status is used when a user is encoding a new and unconfirmed Invoice.\n"
              " * The 'Open' status is used when user creates invoice, an invoice number is generated. It stays in the open status till the user pays the invoice.\n"
+             " * The 'In Payment' status is used when this invoice has been paid in a journal configured to post entries at bank reconciliation and the payment has not yet been reconciled with bank statement\n"
```

Comment:
The 'In Payment' status is used when payments have been registered in a journal configured to post entries at bank reconciliation only, and some of the payments haven't yet been reconciled with a bank statement line